posthog-core/src/types.ts

```diff
@@ -52,6 +52,8 @@ export type PostHogCoreOptions = {
   sessionExpirationTimeSeconds?: number
   /** Whether to post events to PostHog in JSON or compressed format. Defaults to 'json' */
   captureMode?: 'json' | 'form'
+  /** Whether to disable GZIP compression */
+  disableCompression?: boolean
```

Comment:
either this flag should consider lz64 or we should rename to be specifically about gzip compression

Reply:
nvm, `lz64` is deprecated and will be removed at some point